lib/yggdrasil/command/move.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Yggdrasil.Command.MoveCommand do
+  defstruct exit: nil
+end
+
+defimpl Yggdrasil.Command, for: Yggdrasil.Command.MoveCommand do
```

Comment:
Btw, this can be inside the module I think that is more idiomatic.

```elixir
defmodule Yggdrasil.Command.MoveCommand do
  defstruct exit: nil

  defimpl Yggdrasil.Command do
    alias Yggdrasil.Message

    def execute(cmd, player) do
      Yggdrasil.Player.notify player, Message.info("You head towards #{cmd.exit}")
    end
  end
end
```